specs/processes.table

```diff
@@ -32,6 +32,9 @@ schema([
 ])
 extended_schema(WINDOWS, [
     Column("is_elevated_token", INTEGER, "Process uses elevated token yes=1, no=0"),
+    Column("is_secure_process", INTEGER, "Process is secure (IUM) yes=1, no=0"),
```

Comment:
Nitpick and for general consistency I prefer if we avoid the `is_` prefix for column names. This is because the column type implies the verb. Could we name this `secure_process`, etc?

We can change the `is_elevated_token` to just `elevated_token` as well?

Reply:
I was trying to be consistent following `is_elevated_token` 😄 
I've changed them all as suggested now.